SPECS/k3s/k3s.spec

```diff
@@ -82,9 +82,10 @@ exit 0
 %{install_sh}
 
 %changelog
-* Wed Apr 05 2023 Anuj Garg <anujgarg@microsoft.com> - 1.26.2-1
+* Wed Apr 05 2023 Anuj Garg <anujgarg@microsoft.com> - 1.26.2-2
```

Comment:
Please add a new entry.
We do not modify an old entry.

Reply:
Done